pkg/quota/generic/evaluator.go

```diff
@@ -81,7 +81,8 @@ func MatchesNoScopeFunc(scope api.ResourceQuotaScope, object runtime.Object) (bo
 }
 
 // Matches returns true if the quota matches the specified item.
-func Matches(resourceQuota *api.ResourceQuota, item runtime.Object, matchFunc MatchingResourceNamesFunc, scopeFunc MatchesScopeFunc) (bool, error) {
+func Matches(resourceQuota *api.ResourceQuota, item runtime.Object,
+	matchFunc MatchingResourceNamesFunc, scopeFunc MatchesScopeFunc) (bool, error) {
```

Comment:
nit: is it intentional?

Reply:
@aveshagarwal yes. for breaking very long line into two.